app/views/posts/show.html.erb

```diff
@@ -1,2 +1,2 @@
 <h1>Posts#show</h1>
-<p>Find me in app/views/posts/show.html.erb</p>
+<%= render partial: 'biopost', locals: { post: @post } %>
```

Comment:
Nice job making use of ruby partials, as this makes your codebase maintainable 😉. Kindly ensure to stick to the [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png) and ensure all comments for a post are rendered in the post show page👍🏼.